protoc-gen-swagger/BUILD.bazel

```diff
@@ -19,6 +19,7 @@ go_library(
 go_binary(
     name = "protoc-gen-swagger",
     embed = [":go_default_library"],
+    visibility = ["//visibility:public"],
```

Comment:
Not sure why you need to set the visibility to be public here.

Reply:
This binary is now an implicit dependency of protoc_gen_swagger since it is listed as the default label for the _protoc_gen_swagger attribute of that rule so it needs to be public since protoc_gen_swagger is itself publicly exposed